hack/addheaders.sh

```diff
@@ -15,6 +15,7 @@
 
 if ! hash addlicense 2>/dev/null; then
   go get -u github.com/google/addlicense
```

Comment:
we can probably remove go get and keep only the go install this way go.mod don't gets updated.

we can use: 
```bash
go install github.com/google/addlicense@latest
```

or use another specific version.